lib/microservice-call.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
-const { promisify } = require('util');
-const request = promisify(require('request'));
 const RouterFetcher = require('@janiscommerce/router-fetcher');
+const axios = require('axios');
```

Comment:
Agreguemos el `.default`

```suggestion
const axios = require('axios').default;
```

Para traer todos los TypeDef de Axios